packages/app-content-pages/README.md

```diff
@@ -25,11 +25,8 @@ To get the credentials, go to https://app.contentful.com/spaces/jt90kyhvp0qv/api
 
 - `docker-compose up` to run a server on http://localhost:3000.
 - `docker-compose down` to stop the dev server.
-- `docker-compose run --rm dev test` to run the tests.
-
-##### Preview the production Dockerfile image build
-
-- `docker-compose -f docker-compose-prod.yml up` to run a production Dockerfile image server on http://localhost:3000.
+- `docker-compose run --rm content-pages test` to run the tests.
+- `docker-compose --rm content-pages start` to run a production build on http://localhost:3000.
```

Comment:
not blocking at all - I guess the production version will depend on the final setup of the env vars there could be another service in the compose file (or compose file) for explicit prod / dev builds.

Reply:
Move the dev services up into the top-level compose file? That would keep everything in one place. I don't know how people feel about having package-level compose files vs. a single compose file for everything in the monorepo. Currently there are dev services, spread across the different packages, for content-pages, project pages, the dev classifier and all the storybooks (including Zooniverse React Components.)

I've made the assumption that if you're using Docker for an individual package, you're doing development work on that package. If you're using Docker at the top level, it's because you want to start all the services and see how they'll work together in production.